docs/guides/integration-prebidjs.md

```diff
@@ -0,0 +1,193 @@
+
+See [Normalization and Encoding](../getting-started/gs-normalization-encoding.md) for more details.
+:::
+
+:::tip
+The UID2 module will ensure that all DII is hashed and encrypted before sending it to the UID2 service.
+:::
+
+### Configure for email
+
+Configure the UID2 module with an email address:
+
+```js
+pbjs.setConfig({
+  userSync: {
+    userIds: [{
+      name: 'uid2',
+      params: {
+        serverPublicKey: '...server public key...',
+        subscriptionId: '...subcription id...',
+        email: 'user@example.com',
+      }
+    }]
+  }
+});
+```
+
+The UID2 module will normalize and hash the email address before sending the encrypted hash to the UID2 service.
+
+### Configure for phone
+
+Configure the UID2 module with a phone number:
+
+```js
+pbjs.setConfig({
+  userSync: {
+    userIds: [{
+      name: 'uid2',
+      params: {
+        serverPublicKey: '...server public key...',
+        subscriptionId: '...subcription id...',
+        phone: '+1111111111',
+      }
+    }]
+  }
+});
+```
+
+The UID2 module will hash the phone number before sending the encrypted hash to the UID2 service.
+
+The publisher is responsible for normalizing the phone number. Refer to [Normalization and Encoding](../getting-started/gs-normalization-encoding.md#phone-number-normalization) for details on phone number normalization.
+
+### Configure for hashed email
+
+Configure the UID2 module with a hashed email address:
+
+```js
+pbjs.setConfig({
+  userSync: {
+    userIds: [{
+      name: 'uid2',
+      params: {
+        serverPublicKey: '...server public key...',
+        subscriptionId: '...subcription id...',
+        emailHash: 'eVvLS/Vg+YZ6+z3i0NOpSXYyQAfEXqCZ7BTpAjFUBUc=',
+      }
+    }]
+  }
+});
+```
+
+The UID2 module will encrypt the hash before sending it to the UID2 service.
+
+The publisher is responsible for normalizing and hashing the email address. Refer to [Normalization and Encoding](../getting-started/gs-normalization-encoding.md) for details on email address normalization and hashing.
+
+### Configure for hashed phone
+
+Configure the UID2 module with a hashed phone number:
+
+```js
+pbjs.setConfig({
+  userSync: {
+    userIds: [{
+      name: 'uid2',
+      params: {
+        serverPublicKey: '...server public key...',
+        subscriptionId: '...subcription id...',
+        phoneHash: 'eVvLS/Vg+YZ6+z3i0NOpSXYyQAfEXqCZ7BTpAjFUBUc=',
+      }
+    }]
+  }
+});
+```
+
+The UID2 module will encrypt the hash before sending it to the UID2 service.
+
+The publisher is responsible for normalizing and hashing the phone number. Refer to [Normalization and Encoding](../getting-started/gs-normalization-encoding.md) for details on phone number normalization and hashing.
+
+## Check the integration
+
+Check that Prebid.js and the UID2 module are configured correctly by calling `pbjs.getUserIds().uid2`. If everything is working, this will return the UID2 token.
```

Comment:
Is it helpful to mention the Professor Prebid and screencap you wrote [previously](https://atlassian.thetradedesk.com/confluence/display/UID2/UID2+CSTG+Integration+Instructions#UID2CSTGIntegrationInstructions-CheckingPrebid.js) here?

Reply:
I might just link to Prebid's pages on [Troubleshooting](https://docs.prebid.org/troubleshooting/troubleshooting-guide.html) and
[Debugging Prebid.js](https://docs.prebid.org/debugging/debugging.html) which mentions Professor Prebid. I don't want to take on the task of documenting a lot of Prebid specific information here.

Reply:
ok sounds good